corehq/motech/dhis2/static/dhis2/js/dataset_map.js

```diff
@@ -20,6 +22,10 @@ hqDefine("dhis2/js/dataset_map", [
                 $sendNowResult.addClass("text-danger");
             }
             $sendNowResult.text(gettext('DHIS2 response: ') + response.text);
+            if (response.responseJSON) {
+                $remoteLogsLink.attr('href', response.responseJSON['log_url']);
```

Comment:
I see you are both changing this attribute with a knockout `data-bind` and here directly with jQuery. I think the preference is that this is handled through knockout if possible. I think you'd need to save this `log_url` to the observable property (you used the name `hrefUrl` in the template). 

I'd also then remove all the `.hide()` and `.show` and use knockout's `visible` binding to control that.

Reply:
@proteusvacuum  I initially went for using `ko`, but then decided to just using jQuery since the rest of this particular file only used jQuery and having 2 ways of "doing the same thing" might be confusing for future eyes. (The  `data-bind` is a remnant of my initial implementation.)

Reply:
@proteusvacuum At the moment I stuck with just jQuery. However, if it's like a definite preference to rather go with `ko`, I don't mind refactoring the file. Feel free to express some comments

Reply:
Good rationale, thanks!